src/mux-analytics.brs

```diff
@@ -803,7 +794,8 @@ function muxAnalytics() as Object
     props.player_is_fullscreen = m.PLAYER_IS_FULLSCREEN
     props.beacon_domain = m._getDomain(m.beaconUrl)
 
-    props.player_instance_id = m._generateShortID()
+    ' We are moving towards using GUID style instance IDs
+    props.player_instance_id = m._generateViewID()
```

Comment:
I see we already have `_generateViewID` defined in this SDK. Wondering why it was not used from the beginning?

Reply:
Originally all of our instance IDs were shorter almost unique IDs (but not at all unique). We decided to go with GUID style ones as of API VERSION 2.1, so that we can "guarantee" uniqueness in the collectors/processors

Reply:
Right, I remember when we made that change. I was more curious about why `_generateViewID` was already an existing function and not something we had to add in this PR.

That's all. Looks good.